hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl

```
@@ -1062,8 +1062,18 @@ objects:
         annotations:
           openshift.io/node-selector: ''
         labels:
+          managed.openshift.io/service-lb-quota-exempt: "true"
+          managed.openshift.io/storage-pv-quota-exempt: "true"
           openshift.io/cluster-logging: 'true'
           openshift.io/cluster-monitoring: 'true'
+    - apiVersion: v1
+      kind: ResourceQuota
+      metadata:
+        name: logging-storage-quota
+        namespace: openshift-logging
+      spec:
+        hard:
+          requests.storage: "600G"
```

```suggestion
          requests.storage: "600Gi"
```

Same as before about units.